### PR TITLE
DOC: Fix broken URLs

### LINKED ISSUE #886: Broken links in docstrings
Typically this one :
http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg
I'm wondering if we can have a tool that checks automatically all links.

### PR DESCRIPTION
**Pull request by mrahim**
Fix #886 

@lesteve : I did a recursive find and replace, something like `find . -type f -print0 | xargs -0 sed -i 's#URL1#URL2#g'`

### CODE REVIEW

**Comment by lesteve** [comment]
Thanks a lot for this, there is still a manipulating_mr_images in nilearn/_utils/exceptions.py.

File: nilearn/plotting/img_plotting.py [hunk 1]
```diff
@@ -967,7 +967,7 @@ def plot_glass_brain(stat_map_img,
             absolute value will be used (rendering positive and negative
             values in the same manner). If set to false the sign of the
             maximum intensity will be represented with different colors.
-            See http://nilearn.github.io/auto_examples/manipulating_visualizing/plot_demo_glass_brain_extensive.html
+            See http://nilearn.github.io/auto_examples/manipulating_visualizing/plot_demo_glass_brain.html
```
Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: You want to keep the glass_brain_extensive here, which will exist once we release and update the website.

Comment (lesteve) [inline nilearn/plotting/img_plotting.py]: For example, you can see that it exists on CircleCI: https://circle-artifacts.com/gh/nilearn/nilearn/827/artifacts/0/home/ubuntu/nilearn/doc/_build/html/auto_examples/manipulating_visualizing/plot_demo_glass_brain_extensive.html

**Comment by lesteve** [comment]
Yeah CircleCI has completed finally and it is green.

Merging, thanks a lot for thinking of it before the release!

Out of interest, how did you spot the broken links not related to niimg?

**Comment by mrahim** [comment]
`make linkcheck`, thanks to @GaelVaroquaux for the advice :)

**Comment by lesteve** [comment]
> make linkcheck

OK that's what I thought but I was just checking.

**Pull request merged**